src/main/java/pl/piekoszek/gorskimatches/equation/SolvableEquations.java

```diff
@@ -2,29 +2,16 @@
 
 import org.springframework.stereotype.Component;
 
-import java.util.HashMap;
-import java.util.Map;
-
 @Component
 public class SolvableEquations {
```

Comment:
hmmm I don't like this name :D
maybe sth like
`SolutionChecker` `SolutionValidator` `QuizAnswerChecker`, sth else?